hibernate-core/src/test/java/org/hibernate/jpa/test/transaction/CloseEntityManagerWithActiveTransactionTest.java

```diff
@@ -83,6 +86,23 @@ public void tearDown() throws Exception {
 		}
 	}
 
+	@Test(expected = IllegalStateException.class)
+	@TestForIssue( jiraKey = "HHH-12487")
+	public void getTransactionOnClosedJtaEntityManagerShouldReturnNUll() {
+		EntityManager em = createEntityManager();
+		em.close();
+		em.getTransaction();
+		fail( "Calling getTransaction on a JTA entity manager should throw an IllegalStateException" );
+	}
+
+	@Test(expected = IllegalStateException.class)
+	@TestForIssue( jiraKey = "HHH-12487")
+	public void getTransactionOnAJtaEntityManagerShouldReturnNUll() {
```

Comment:
Same regarding name - this should not return null, it should throw an exception.  The assertions are correct, the method name should change.